client/src/components/ItemSummary.jsx

```diff
@@ -36,34 +36,37 @@ class ItemSummary extends React.Component {
   }
 
   selectColor(event) {
-    console.log(event.target.value);
-    this.setState({ selectedColor: event.target.value })
+    var capitalizedColor = event.target.value.charAt(0).toUpperCase() + event.target.value.slice(1);
+    var parsedColors = JSON.parse(this.state.item[0].colors);
+    var matchingColorObj = parsedColors.find(color => color.colorName === capitalizedColor);
```

Comment:
You might consider getting out of the habit of using abbreviated terms like "Obj", at least for now until you're in an environment where it's the norm.